src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTab.cmp

```diff
@@ -81,6 +82,19 @@
         </div>
     </header>
 
+    <aura:if isTrue="{!v.showRetirementPrompt}">
+        <c:utilPrompt 
+            aura:id="retirementPrompt" 
+            id="retirementPrompt"
+            headerTitle="{!$Label.c.bgeRetirementWarningHeader}"
+            promptMessage="{!$Label.c.bgeRetirementWarningBody}"
+            confirmButtonTitle="Confirm" 
```

Comment:
I'm surprised we don't have a label for the word "Confirm" already. One thing to note is that once BGE is retired, we can deprecate these labels. To that end, should we label this one up too?